etc/skydive.yml.default

```diff
@@ -288,6 +288,9 @@ ovs:
 docker:
   # url: unix:///var/run/docker.sock
 
+libvirt:
```

Comment:
We are in the process of moving the agent probes configuration from the top level.
Can you please move it to `agent.topology.probe.libvirt` ?